deepchem/models/graph_models.py

```diff
@@ -715,7 +715,7 @@ class _GraphConvKerasModel(tf.keras.Model):
   def __init__(self,
                n_tasks,
                graph_conv_layers,
-               dense_layer_size=128,
+               dense_layers, # changed dense_layer_size to dense_layers as a list instead
```

Comment:
Could you add type hints for this class? Check out our coding conventions page which provides an overview of our recent transition to type hints: https://deepchem.readthedocs.io/en/latest/coding.html